### PR TITLE
Create System Recovery Plan Exercise Template.md

### PR DESCRIPTION
**Pull request by joeniquette**

### CODE REVIEW

File: products/identity/Incident Response/System Recovery Plan Exercise Template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Execution
+
+1. Describe the scenario and determine if an incident should be declared.
+    1. Review the definition of an incident document (github link needed) if you are unsure of the qualifications.
+    2. do not declare a real incident at this time, just talk through doing it.
+2. Assign roles and responsibilities to team members for handling the incident.
+3. Describe the process of the service SRP, walk through the steps required
+    1. any identified gaps should be noted for future improvements to the SRP
+4. Conduct a post-exercise review to discuss what went well and what could be improved.
```
Comment (bosawt) [inline products/identity/Incident Response/System Recovery Plan Exercise Template.md:27]: Do we want to define any sort of retrospective documentation or required follow up docs, or is that not necessary for SRP specifically?

Comment (joeniquette) [reply products/identity/Incident Response/System Recovery Plan Exercise Template.md:27]: I just made [this](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Documentation/Internal_Process/retro_template.md), would this work to link it?

Comment (bosawt) [reply products/identity/Incident Response/System Recovery Plan Exercise Template.md]: Yeah

**Review by bosawt — APPROVED** [review]
looks good to me, we can iterate if necessary after going through the mock incident exercises

**Pull request merged**